lib/ddtrace/context_flush.rb

```diff
@@ -26,6 +26,8 @@ class Partial
       # Start flushing partial trace after this many active spans in one trace
       DEFAULT_MIN_SPANS_FOR_PARTIAL_FLUSH = 500
 
+      attr_reader :min_spans_for_partial
+
```

Comment:
Hmmm, is this API being made public only for the tests, or are customers expected to use it as well?

I guess this is one of the challenges we face today -- I think we kinda expose a tad too much that it's not clear if it's just because we need to validate it or because it's useful for customers 😅

Reply:
It's for testing.
I can make it private.
I don't think we have the right tools in Ruby sometimes for testing.

Reply:
I see. I think having it private is a bit better -- it's sometimes "dirtier" and a bit of a code smell to be testing private methods directly, but between that and clearly signaling what customers should and shouldn't use, I think the latter ends up being more important.

> I don't think we have the right tools in Ruby sometimes for testing.

Agreed.